### DISCUSSION TITLE
Sunset Photo

### DISCUSSION THREAD

**Discussion by jacobf19** [jakobzhao/geog495#33] [Cartogram]
https://api.mapbox.com/styles/v1/jacobf19/ckvikwksqjer414o7shg4w1ld.html?title=copy&access_token=pk.eyJ1IjoiamFjb2JmMTkiLCJhIjoiY2t2OGU4ZGVpMDB6aTJ1czE2M2t4YXJ5eiJ9.n5sSIOzCDN-IO7ttOYpVxA&zoomwheel=true&fresh=true#9.04/40.8419/-74.0008
![IMG_1153](https://user-images.githubusercontent.com/72268448/139952323-183acd5e-8316-4d08-a2b3-28433ad3e853.JPG)

**Comment by jakobzhao** [comment]
dark blue base color with lively yellow labels,  well designed!
